.github/workflows/subgraph.yml

```diff
@@ -6,7 +6,6 @@ on:
       - main
 
```

Comment:
Do we need the folder filter here?

Reply:
Since we are updating subgraph tests before every release, doing folder filtering raises incompatibility issues between the outdated tests and the new mapping Assembly Script file.
Maybe having folder filtering and the current condition together can work, but still I think for all merges to main, the workflow should run.